sites/github.styl

```diff
@@ -583,10 +609,13 @@ a.label
         color transparentify(green, color-background, 0.65)
 
 .octicon-alert
-.octicon-check
     color white
 .octicon-x
     color red
+.octicon-check
+    color green
```

Comment:
Without having investigated it myself, is this the best way to do this?  I'm guessing that I set `.octicon-check` to white for when it appears on a green background, IIRC.  Do we need to use a more specific selector for when it should be green?

OTOH maybe the CSS has changed and this is now the right way to do it.  :)

Reply:
This is a bit finiky. Github uses the `.text-green` selector to make the checks green. I tried overriding it, but it seemed the `.octicon` general selector was still overriding it. `.octicon-check` was the most specific selector I could find which worked. 

I don't *think* that the check is ever on a green background, I've only seen it on the background, like in branches. 

If there was a way of 'turning off' the `.octicon` custom style just for a class, then we could do it this way (but I don't know if that's possible).

Reply:
Ok, sounds good, thanks!